projects.go

```diff
@@ -67,6 +67,7 @@ type Project struct {
 	ContainerRegistryImagePrefix              string                     `json:"container_registry_image_prefix,omitempty"`
 	CreatedAt                                 *time.Time                 `json:"created_at,omitempty"`
 	LastActivityAt                            *time.Time                 `json:"last_activity_at,omitempty"`
+	UpdatedAt                                 *time.Time                 `json:"updated_at,omitempty"`
```

Comment:
Please place it above `LastActivityAt` following the ordering in the docs...

Reply:
oh, sure, should be changed now :)